scss/components/identifier.scss

```diff
@@ -111,9 +112,53 @@ $block: #{$fd-namespace}-identifier;
     }
     //background
     &--transparent {
-      --fd-identifier-color: var(--fd-color-text-1);
+      --fd-identifier-color: var(--fd-color-text-1, map-get($fd-colors-text, 1));
       --fd-identifier-background-color: transparent;
       @include fd-var-color("color", $fd-color);
       @include fd-var-color("background-color", transparent);
     }
+
+    .#{$fd-namespace}-has-background-color-none {
+      background-color: transparent !important;
+    }
+}
+
+.fd-has-background-color-accent-1 {
+  background-color: var(--fd-color-accent-1, map-get($fd-colors-accent, 1));
+}
+
+.fd-has-background-color-accent-2 {
+  background-color: var(--fd-color-accent-2, map-get($fd-colors-accent, 2));
+}
+
+.fd-has-background-color-accent-3 {
+  background-color: var(--fd-color-accent-3, map-get($fd-colors-accent, 3));
+}
+
+.fd-has-background-color-accent-4 {
+  background-color: var(--fd-color-accent-4, map-get($fd-colors-accent, 4));
+}
+
+.fd-has-background-color-accent-5 {
+  background-color: var(--fd-color-accent-5, map-get($fd-colors-accent, 5));
+}
+
+.fd-has-background-color-accent-6 {
+  background-color: var(--fd-color-accent-6, map-get($fd-colors-accent, 6));
+}
+
+.fd-has-background-color-accent-7 {
+  background-color: var(--fd-color-accent-7, map-get($fd-colors-accent, 7));
+}
+
+.fd-has-background-color-accent-8 {
+  background-color: var(--fd-color-accent-8, map-get($fd-colors-accent, 8));
+}
+
+.fd-has-background-color-accent-9 {
+  background-color: var(--fd-color-accent-9, map-get($fd-colors-accent, 9));
+}
+
+.fd-has-background-color-accent-12 {
+  background-color: var(--fd-color-accent-12, map-get($fd-colors-accent, 12));
```

Comment:
Where is accent 12 coming from?  I only see 9 in the playground and docs.  And if there is a "12", what about "10" and "11"?

Reply:
accent 12 is the color for this at Spec section:
![Screen Shot 2019-07-11 at 9 23 12 AM](https://user-images.githubusercontent.com/41170611/61054504-94aca680-a3bd-11e9-8158-5a7f29e2f197.png)